app/views/layouts/blacklight/base.html.erb

```diff
@@ -33,6 +33,49 @@
       More info: https://getbootstrap.com/docs/4.6/components/popovers/
     -->
     <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"></script>
+    <script type ="application/ld+json">
+     <% if !@document.nil?%> 
+      { "@context": "http://schema.org",
+        "@type": "Dataset",
+        "identifier": "<%= @document.doi_url %>",
+        "name": "<%= @document.title %>",
+        "author": 
+          [
+          <% @document.authors.each do |author| %>
+            { 
+              "name": "<%= author %>"
+              "affiliation":
+              "identifier": 
+            }
+            <% end %>
+          ],
+          "datePublished": "<%= @document.date_created %>",
+          "dateModified": "<%= @document.date_modified %>",
+          "version": "", 
+          "description": "<%= @document.description %>",
+          "keywords": 
+          [
+          ],
+          "citation:": 
+          [
+            <%# "<%= @document.citation % >", %>
+            "",
+          ],
+          "schemaVersion": "https://schema.org/version/28.0",
+          "license": 
+          {
+            "@type": "Dataset",
+            "text": "",
+            "url": "",
+          },
+          "provider": 
+          {
+              "@type": "Organization",
+              "name": "PDC Discovery"
```

Comment:
Pretty sure this organization should be "Princeton University," but let's ask PRDS. @matthewjchandler or @sec122 What do you think?

Reply:
Yes, the organization should be "Princeton University"